src/components/crossDevice/ClientSessionLinked/index.tsx

```diff
@@ -0,0 +1,69 @@
+import { h, FunctionComponent } from 'preact'
+import classNames from 'classnames'
+import { Button } from '@onfido/castor-react'
+
+import ScreenLayout from '../../Theme/ScreenLayout'
+import PageTitle from '../../PageTitle'
+import { trackComponent } from '../../../Tracker'
+import { useLocales } from '~locales'
+import theme from '../../Theme/style.scss'
+import style from './style.scss'
+
+import type { StepComponentBaseProps } from '~types/routers'
+
+type Props = {
+  nextStep: () => void
+} & StepComponentBaseProps
+
+const ClientSessionLinked: FunctionComponent<Props> = ({ nextStep }) => {
```

Comment:
By looking at the `StepComponentMap` I figured out that this component will not be shown as soon as the user opens the cross-device tab. As mentioned in another comment, I think this might be quite confusing for the user and looks like it's adding a bit of complexity on our side too.

Reply:
> I would rename this component to `CrossDeviceClientIntro` because it's always shown as soon as you open the cross-device tab, right?
Totally! I was just thinking when working on the follow on ticket to this that the component needed a rename to something like what you've suggested to make it clearer what it is.

Reply:
> By looking at the `StepComponentMap` I figured out that this component will not be shown as soon as the user opens the cross-device tab. As mentioned in another comment, I think this might be easily missed by the user and looks like it's adding a bit of complexity on our side too.

It actually is. Or rather, it is the first screen the end user sees on loading the Cross Device link on their device browser and the user will need to tap on the "Continue" button on the screen to continue their cross device capture flow...unless I've misunderstood what you meant here.